Backpack-SwiftUI/AppSearchModal/Classes/BPKAppSearchModal.swift

```diff
@@ -22,20 +22,22 @@ public struct BPKAppSearchModal: View {
     let title: String
     @Binding var inputText: String
     let inputPrefix: BPKSearchInputSummary.InputPrefix
+    let clearAction: BPKSearchInputSummary.ClearAction?
     let inputHint: String
     let results: BPKAppSearchModalResults
     let closeAccessibilityLabel: String
     let onClose: () -> Void
-    private var textFieldState: TextFieldState = .default
+    @State private var textFieldState: TextFieldState = .default
```

Comment:
we should remove this property, its no longer used

Reply:
It's removed yes